### PR TITLE
🔨(lti) rework lti development view

### PR DESCRIPTION
**Pull request by wilbrdt**
## Purpose

We could not change LTI fields value without changing them in the .env and refreshing the development page.

## Proposal

Reworking LTI development view so we can change some fields, refresh the signature and POST the LTI launch request to the iframe.

Screenshot from 2024-04-26 12-05-30

### CODE REVIEW

File: src/app/apps/development/templates/development/lti_development.html [hunk 1]
```diff
@@ -10,14 +10,22 @@
         width: 100%;
         height: 80vh;
       }
-
+      input:read-write, select {
+            background-color: rgb(38, 58, 66);
+         }
```
Comment (kernicPanel) [inline src/app/apps/development/templates/development/lti_development.html:15]:
```suggestion
      input:read-write, select {
        background-color: rgb(38, 58, 66);
      }
```

File: src/app/apps/development/templates/development/lti_development.html [hunk 2]
```diff
@@ -10,14 +10,22 @@
...
     <main class="container-fluid">
       <section>
         <h1>LTI Consumer test</h1>
-        <form id="lti_request" method="post" action="{% url "lti:lti-request-view" "demo" %}" target="lti_request">
+        <form 
+          id="lti_params"
+          method="post"
+          target="lti_request"
+          action="{% url "lti:lti-request-view" "demo" %}"
+          >
```
Comment (kernicPanel) [inline src/app/apps/development/templates/development/lti_development.html:28]:
```suggestion
        >
```

File: src/app/apps/development/templates/development/lti_development.html [hunk 3]
```diff
@@ -145,7 +166,7 @@ <h1>LTI Consumer test</h1>
           {% endif %}
           {% endfor %}
 
-          <input type="submit" />
+          <input type="submit" value="Post" />
```
Comment (kernicPanel) [inline src/app/apps/development/templates/development/lti_development.html:169]:
```suggestion
          <input type="submit" value="Post" />
```

**Review by kernicPanel — APPROVED** [review]
The whitespace police has come

**Comment by wilbrdt** [comment]
@kernicPanel now that I enabled my IDE HTML formatting, it should be a bit better!

File: src/app/apps/development/templates/development/lti_development.html [hunk 4]
```diff
@@ -1,167 +1,129 @@
...
+  <main class="container-fluid">
+    <section>
+      <h1>LTI Consumer test</h1>
+      <form id="lti_params" method="post" target="lti_request" action="{% url " lti:lti-request-view" "demo" %}">
+
+        <div class="grid">
+
+          <div class="input-group">
+            <label for="lti_message_type"><code>lti_message_type</code></label>
+            <input id="lti_message_type" type="text" name="lti_message_type" value="{{ lti_message_type }}" />
```
Comment (kernicPanel) [inline src/app/apps/development/templates/development/lti_development.html:32]: Just a personal preference, but I prefer keeping line wraps.
Also, git prefers it too.

**Review by kernicPanel — APPROVED** [review]

**Pull request merged**